src/main/java/lotto/controller/LottoController.java

```diff
@@ -1,25 +1,20 @@
 package lotto.controller;
 
 import lotto.domain.LottoBuyInfo;
-import lotto.domain.ManualLottos;
 import lotto.domain.Lottos;
-import lotto.domain.WinningLotto;
-import lotto.domain.Lotto;
-import lotto.domain.LottoNumber;
 import lotto.domain.LottoResult;
+import lotto.service.LottoService;
 import lotto.view.LottoInputView;
 import lotto.view.LottoResultView;
 
 public class LottoController {
+
     public static void run() {
         LottoBuyInfo buyInfo = LottoInputView.inputBuyInfo();
-        ManualLottos manualLottos = LottoInputView.inputManualLottoNumbers(buyInfo.getManualCount());
-        Lottos lottos = Lottos.from(buyInfo, manualLottos);
+        Lottos lottos = LottoService.purchaseLotto(buyInfo);
         LottoResultView.showLottos(buyInfo, lottos);
 
-        WinningLotto winningLotto = new WinningLotto(Lotto.from(LottoInputView.inputWinningNumbers()), LottoNumber.from(LottoInputView.inputBonusNumber()));
-        LottoResult result = lottos.match(winningLotto);
-
+        LottoResult result = lottos.match(LottoService.winningLotto());
```

Comment:
```suggestion
        LottoResult result = lottoService.executeStatistics(lottos, buyInfo);
```

당첨결과 분석등과 같은 로직도 서비스로 옮겨도 될 것 같네요!